packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.types.ts

```diff
@@ -41,6 +41,7 @@ export interface ICommandBarProps extends React.HTMLAttributes<HTMLDivElement> {
 
   /**
    * Props to be passed to overflow button

```

Comment:
please end these jsdoc with periods :) You might want to create an example demonstrating this specific scenario as well.

Reply:
oh good call! Yup.